java/ql/lib/semmle/code/java/dataflow/FlowSteps.qll

```diff
@@ -22,6 +22,7 @@ private module Frameworks {
   private import semmle.code.java.frameworks.IoJsonWebToken
   private import semmle.code.java.frameworks.jackson.JacksonSerializability
   private import semmle.code.java.frameworks.InputStream
+  private import semmle.code.java.frameworks.Networking
```

Comment:
Is this import needed? I added it due to a [prior discussion](https://github.com/github/codeql/pull/12852#discussion_r1182438832) that implied a bidirectional import is necessary for this code to work, but my test cases passed without it. 🤔